core/cloudflow-blueprint/src/main/scala/cloudflow/blueprint/deployment/ApplicationDescriptor.scala

```diff
@@ -152,18 +151,17 @@ final case class StreamletDeployment(
 )
 
 object StreamletDeployment {
-  val ServerAttributeName          = "server"
-  val MinimumEndpointContainerPort = 3000
+  val ServerAttributeName   = "server"
+  val EndpointContainerPort = 3000
```

Comment:
What happens when we have multiple end points ? e.g. 2 ingresses especially for local running ..

Reply:
@debasishg see below. The `LocalRunner` is fixed to deal with this directly.

Reply:
👍 .. missed it ..